cpmpy/solvers/choco.py

```diff
@@ -592,7 +592,7 @@ def _get_constraint(self, cpm_expr):
                 return self.chc_model.int_value_precede_chain(self._to_vars(cpm_expr.args[0]), cpm_expr.args[1])
             elif cpm_expr.name == "gcc":
                 vars, vals, occ = cpm_expr.args
-                return self.chc_model.global_cardinality(*self.solver_vars([vars, vals]), self._to_vars(occ), cpm_expr.closed)
+                return self.chc_model.global_cardinality(self._to_vars(vars), vals, self._to_vars(occ), cpm_expr.closed)
```

Comment:
The solver_vars also converts numpy to int, so you need to keep that part for `vals` : )